cfpb/consumerfinance.gov
# Set unique research hub category

Recently, a new "Research Hub" filterable page was created that was intended to filter based on specific unique categories. Based on feedback from the requesting stakeholder and some internal testing, it seems this filter does not currently work properly because the two new Research categories ("Industry and markets" and "Data point") have identical IDs to those equivalents under the standard Report categories. Attempting to modify the ID within Wagtail fails, presumably because Wagtail sees either one as equivalent

Regardless, this PR addresses the stakeholder desire for this page to house specific and unique pages within the Research & Reports section by creating a unique category for such pages: "Office of Research Report". This will need to be applied to pages manually after being created and updated, but the stakeholder has already provided the pages where this needs to be done, so it should be straightforward.

This will remain a draft PR until the stakeholder confirms that the name chosen for the new category is approved on their end.

---




## Additions

- New category under "Research Hub" page type: `Office of Research Report`


## Removals

- Existing categories under "Research Hub" page type: `Industry and markets`, `Data point`


## How to test this PR

1. localhost
2. Modify an existing document detail page under Research & Reports to the category "Office of Research Report", then publish
3. Confirm that said page now appears on both the Research & Reports (/research-reports/) _and_ Research Hub (/data-research/research-hub/) pages



## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
